app/build.gradle

```diff
@@ -48,9 +48,10 @@ dependencies {
     implementation 'org.osmdroid:osmdroid-mapsforge:6.1.10'
     implementation 'org.osmdroid:osmdroid-geopackage:6.1.10'
     implementation 'com.github.yuriy-budiyev:code-scanner:2.1.2'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
why do we this legacy library?

Reply:
I have no idea why that's there. Also, there isn't a textView2 anywhere